### ISSUE TITLE
Not Working in 14 April

### ISSUE THREAD

**Issue by iHafez** [bwmsdroid/exitlag-hwid-bypass#4]
i tried to use it today but its not working sadly :(

**Comment by bwmsdroid** [comment]
Hey!
It is true, it has an issue.
You must execute the program and login and logout until it work.
I am working on a fix for this issue.

**Comment by iHafez** [comment]
yeah but I login and logout 10 times it didnt work :(

**Comment by iHafez** [comment]
like they are patching it or what ?

**Comment by itsOwen** [comment]
> like they are patching it or what ?

Please be patient it will be fixed,

**Comment by iHafez** [comment]
okaay

**Comment by sonatta09** [comment]
yup its normal.. i waited atleast 30-45mins after you registered then do logout and login couple of times in the app... boom it works 3days trial again working

**Comment by bwmsdroid** [comment]
> 
> 
> like they are patching it or what ?

No, they didn't.
It is a issue due an update.
Don't worry, I am working on a fix.
I gonna use other method, I gonna hook some functions used to get the hwid's and return random values, this will fix 99% of the problems.
I am also working on a way to activate the account via websocks, so it  won't be necessary to download a file, I would have a server just to activate exitlag accounts. 
Maybe add some ads on the website activator, so it would pay itself.

IDK, I will focus on the hook thing first.
Just wait.

**Comment by bwmsdroid** [comment]
Hey guys!
It should be fixed!
https://github.com/bwmsdroid/exitlag-hwid-bypass/releases/tag/1.6
Please, give me a feedback.

**Comment by Saintcz** [comment]
> Hey guys!
> It should be fixed!
> https://github.com/bwmsdroid/exitlag-hwid-bypass/releases/tag/1.6
> Please, give me a feedback.

![exitlag](https://user-images.githubusercontent.com/17323765/115102282-31648680-9f4a-11eb-91d1-0e7a0490bc6c.png)

Working! Good job

**Comment by bwmsdroid** [comment]
LOL bro.
I am not sure if it is fixed yet.
I got some unexpected behaviors now.
I will still working on the case.

**Comment by Dubaai** [comment]
bro do you already know about this error?
![dsadsad](https://user-images.githubusercontent.com/63336110/115103576-3fee7600-9f29-11eb-9acd-9eea90462e15.PNG)

**Comment by bwmsdroid** [comment]
> 
> 
> bro do you already know about this error?
> ![dsadsad](https://user-images.githubusercontent.com/63336110/115103576-3fee7600-9f29-11eb-9acd-9eea90462e15.PNG)
> 
> it could be that and I don't understand

Yeah...
I made this logs to better understand the issues.
Please create a new issue, giving me all the information (winver, exitlag version, bypass version, etc.).

**Comment by bwmsdroid** [comment]
> 
> 
> > Hey guys!
> > It should be fixed!
> > https://github.com/bwmsdroid/exitlag-hwid-bypass/releases/tag/1.6
> > Please, give me a feedback.
> 
> ![exitlag](https://user-images.githubusercontent.com/17323765/115102282-31648680-9f4a-11eb-91d1-0e7a0490bc6c.png)
> 
> Working! Good job

OK guys, I've found the issue.
There is one more place that I need to hook, because it is sending processor id and some others hwid.
I just need to code now xD.

Also, I am pretty close to understand how the web socket communication with the server happens.
I alredy know that it is split by 

```
32 DA 01 1A 40 43 41 33 31 33 39 37 31 34 37 46 | 2ڠ @CA31397147F
46 46 33 34 30 43 42 38 36 44 45 32 42 35 37 35 | FF340CB86DE2B575
46 44 32 32 38 30 31 43 39 34 38 36 36 39 35 37 | FD22801C94866957
35 39 43 41 41 32 30 42 34 42 45 46 32 36 32 43 | 59CAA20B4BEF262C
42 37 30 32 46 22 10 42 46 45 42 46 42 46 46 30 | B702F" BFEBFBFF0
30 30 33 30 36 43 33 2A 02 70 74 3A 06 34 2E 39 | 00306C3* pt: 4.9
35 2E 32 42 1E 57 69 6E 64 6F 77 73 20 31 30 20 | 5.2B Windows 10 
56 65 72 73 69 6F 6E 20 32 30 30 39 20 36 34 2D | Version 2009 64-
62 69 74 4A 10 57 69 6E 64 6F 77 73 20 44 65 66 | bitJ Windows Def
65 6E 64 65 72 52 00 58 00 62 40 39 63 32 30 34 | enderR X b@9c204
62 63 37 32 65 32 65 38 64 39 37 34 63 35 66 30 | bc72e2e8d974c5f0
34 35 63 62 34 34 62 39 35 65 38 37 38 64 62 62 | 45cb44b95e878dbb
32 39 33 35 63 30 38 39 33 31 30 66 63 32 30 35 | 2935c089310fc205
64 36 37 63 65 64 66 30 66 65 30 68 00 40 05    | d67cedf0fe0h @  

```
The in formations are splits by `@`, where `magicbytes@hwids@hash_email_password`. 
I just need to understand how it makes the hash of the email and password (both are one).

If someone is up to help me, send me a message on discord.

**Comment by Nathan6969420** [comment]
when opening the bypass it gives me this error 
![image](https://user-images.githubusercontent.com/82702106/115107252-c1222900-9f7a-11eb-858e-a1884d41e7cf.png)
![image](https://user-images.githubusercontent.com/82702106/115107267-d8f9ad00-9f7a-11eb-9719-b7a8f41cd73f.png)

**Comment by iHafez** [comment]
its not giving me any error, it tells me success but its not working well... I have no VIP days.
waiting for version 1.7

**Comment by noacx** [comment]
OK OK , I AM WAITING FOR THE FIX......BUT HATS OFF FOR YOU TO DO THIS  :)

**Comment by noacx** [comment]
> > > Hey guys!
> > > It should be fixed!
> > > https://github.com/bwmsdroid/exitlag-hwid-bypass/releases/tag/1.6
> > > Please, give me a feedback.
> > 
> > 
> > ![exitlag](https://user-images.githubusercontent.com/17323765/115102282-31648680-9f4a-11eb-91d1-0e7a0490bc6c.png)
> > Working! Good job
> 
> OK guys, I've found the issue.
> There is one more place that I need to hook, because it is sending processor id and some others hwid.
> I just need to code now xD.
> 
> Also, I am pretty close to understand how the web socket communication with the server happens.
> I alredy know that it is split by
> 
> ```
> 32 DA 01 1A 40 43 41 33 31 33 39 37 31 34 37 46 | 2ڠ @CA31397147F
> 46 46 33 34 30 43 42 38 36 44 45 32 42 35 37 35 | FF340CB86DE2B575
> 46 44 32 32 38 30 31 43 39 34 38 36 36 39 35 37 | FD22801C94866957
> 35 39 43 41 41 32 30 42 34 42 45 46 32 36 32 43 | 59CAA20B4BEF262C
> 42 37 30 32 46 22 10 42 46 45 42 46 42 46 46 30 | B702F" BFEBFBFF0
> 30 30 33 30 36 43 33 2A 02 70 74 3A 06 34 2E 39 | 00306C3* pt: 4.9
> 35 2E 32 42 1E 57 69 6E 64 6F 77 73 20 31 30 20 | 5.2B Windows 10 
> 56 65 72 73 69 6F 6E 20 32 30 30 39 20 36 34 2D | Version 2009 64-
> 62 69 74 4A 10 57 69 6E 64 6F 77 73 20 44 65 66 | bitJ Windows Def
> 65 6E 64 65 72 52 00 58 00 62 40 39 63 32 30 34 | enderR X b@9c204
> 62 63 37 32 65 32 65 38 64 39 37 34 63 35 66 30 | bc72e2e8d974c5f0
> 34 35 63 62 34 34 62 39 35 65 38 37 38 64 62 62 | 45cb44b95e878dbb
> 32 39 33 35 63 30 38 39 33 31 30 66 63 32 30 35 | 2935c089310fc205
> 64 36 37 63 65 64 66 30 66 65 30 68 00 40 05    | d67cedf0fe0h @  
> ```
> 
> The in formations are splits by `@`, where `magicbytes@hwids@hash_email_password`.
> I just need to understand how it makes the hash of the email and password (both are one).
> 
> If someone is up to help me, send me a message on discord.

VERSION 1.3 IS WORKING IDK WHY :)

**Comment by iHafez** [comment]
VERSION 1.3 is not working for me

**Comment by bwmsdroid** [comment]
Of course version 1.3 doesn't work.

**Comment by bwmsdroid** [comment]
> 
> 
> > > > Hey guys!
> > > > It should be fixed!
> > > > https://github.com/bwmsdroid/exitlag-hwid-bypass/releases/tag/1.6
> > > > Please, give me a feedback.
> > > 
> > > 
> > > ![exitlag](https://user-images.githubusercontent.com/17323765/115102282-31648680-9f4a-11eb-91d1-0e7a0490bc6c.png)
> > > Working! Good job
> > 
> > 
> > OK guys, I've found the issue.
> > There is one more place that I need to hook, because it is sending processor id and some others hwid.
> > I just need to code now xD.
> > Also, I am pretty close to understand how the web socket communication with the server happens.
> > I alredy know that it is split by
> > ```
> > 32 DA 01 1A 40 43 41 33 31 33 39 37 31 34 37 46 | 2ڠ @CA31397147F
> > 46 46 33 34 30 43 42 38 36 44 45 32 42 35 37 35 | FF340CB86DE2B575
> > 46 44 32 32 38 30 31 43 39 34 38 36 36 39 35 37 | FD22801C94866957
> > 35 39 43 41 41 32 30 42 34 42 45 46 32 36 32 43 | 59CAA20B4BEF262C
> > 42 37 30 32 46 22 10 42 46 45 42 46 42 46 46 30 | B702F" BFEBFBFF0
> > 30 30 33 30 36 43 33 2A 02 70 74 3A 06 34 2E 39 | 00306C3* pt: 4.9
> > 35 2E 32 42 1E 57 69 6E 64 6F 77 73 20 31 30 20 | 5.2B Windows 10 
> > 56 65 72 73 69 6F 6E 20 32 30 30 39 20 36 34 2D | Version 2009 64-
> > 62 69 74 4A 10 57 69 6E 64 6F 77 73 20 44 65 66 | bitJ Windows Def
> > 65 6E 64 65 72 52 00 58 00 62 40 39 63 32 30 34 | enderR X b@9c204
> > 62 63 37 32 65 32 65 38 64 39 37 34 63 35 66 30 | bc72e2e8d974c5f0
> > 34 35 63 62 34 34 62 39 35 65 38 37 38 64 62 62 | 45cb44b95e878dbb
> > 32 39 33 35 63 30 38 39 33 31 30 66 63 32 30 35 | 2935c089310fc205
> > 64 36 37 63 65 64 66 30 66 65 30 68 00 40 05    | d67cedf0fe0h @  
> > ```
> > 
> > 
> > The in formations are splits by `@`, where `magicbytes@hwids@hash_email_password`.
> > I just need to understand how it makes the hash of the email and password (both are one).
> > If someone is up to help me, send me a message on discord.
> 
> VERSION 1.3 IS WORKING IDK WHY :)

Nah, it is just a misunderstand. 1.3 has the same issue of 1.7. 
I already know the problem and I am working on a solution.

**Comment by iHafez** [comment]
waiting for version 1.8 ...

**Comment by emrearapoglu** [comment]
It's not working for me.

**Comment by flakel1337** [comment]
bro, no longer works, worked yesterday

**Comment by kasperlino6** [comment]
not working :(

**Comment by bwmsdroid** [comment]
Yep. I made a python script that activate exitlag accounts just by the email and password, open source.
I hosted it on a server and put some google adsense.
I am just waiting for adsense approves my website.

This is a way to avoid **issues of incompatibility**, **so everyone can use**, plus **I am able to monetize** and pay for the cloud that I hosted the server. Also, **you don't need to download** and **it is easier to use**.

My program (python script) does work without any problem, unlike exitlag bypass made in cpp.
It connects direct to the exitlag server and opens a socket connection sending all hte data necessary to create the account, plus it randomize the hwid.

Thank you for understand guys :D

**Comment by bwmsdroid** [comment]
Btw, I made a discord server.
https://discord.gg/2KPYrRFqpY

**Comment by kasperlino6** [comment]
could you tell the website name?

**Comment by bwmsdroid** [comment]
it isn't ready yet :).
I am waiting for good adsense accept it.

**Comment by kasperlino6** [comment]
> it isn't ready yet :).
> I am waiting for good adsense accept it.

oh ok
